phf_shared/src/lib.rs

```diff
@@ -111,6 +111,19 @@ delegate_debug!(u128);
 delegate_debug!(i128);
 delegate_debug!(bool);
 
+delegate_debug!(String);
```

Comment:
For consistency I would probably move this one up above under the `delegate_debug!(str);` line then have this next grouping be for `Vec<>()`. Would it make sense to have `Vec<String>` as well?